EIPS/eip-4910.md

```diff
@@ -0,0 +1,1111 @@
+
+## Specification Outline
+
+This proposal introduces several new concepts as extensions to the ERC721 standard:
+* **Royalty Account (RA)**
+    * A Royalty Account is attached to each NFT through its `tokenId` and consists of several sub-accounts which can be accounts of individuals or other RAs. A Royalty Account is identified by an account identifier.
+* **Account Type**
+    * This specifies if an RA Sub Account belongs to an individual (user) or is another RA. If there is another RA as an RA Sub Account, the allocated balance needs to be reallocated to the Sub Accounts making up the referenced RA.
+* **Royalty Split**
+    * The percentage each Sub Account receives based on a sale of an NFT that is associated with an RA
+* **Royalty Balance**
+    * The royalty balance associated with an RA
+* **Sub Account Royalty Balance**
+    * The royalty balance associated to each RA Sub Account. Note that only individual accounts can carry a balance that can be paid out. That means that if an RA Sub Account is an RA, its final Sub Account balance must be zero, since all RA balances must be allocated to individual accounts. 
+* **Token Type**
+    * Token Type is given as either ETH or the symbol of the supported ERC 20/223/777 tokens such as `DAI`
+* **Asset ID**
+    * This is the `tokenId` the RA belongs to.
+* **Parent**
+    * This indicates which `tokenId` is the immediate parent of the `tokenId` to which an RA belongs.
+
+### Data Structures
+
+In order to create an interconnected data structure linking NFTs to RAs that is search optimized requires the following additions to the global data structures of an ERC721:
+
+* Adding structs for a Royalty Account and associated Royalty Sub Accounts to establish the concept of a Royalty Account with sub accounts.
+* Defining an `raAccountId` as the keccak256 hash of `tokenId`, the actual `owner` address, and the current block number, `block.blocknumber`
+* Mapping a `tokenId` to an `raAccountID` in order to connect an RA `raAccountId` to a `tokenId`
+* Mapping the `raAccountID` to a `RoyaltyAccount` in order to connect the account identifier to the actual account.
+* An `ancestry` mapping of the parent-to-child NFT relationship
+* A mapping of supported token types to their origin contracts and last validated balance (for trading and royalty payment purposes)
+* A mapping with a struct for a registered payment to be made in the `executePayment` function and validated in `safeTransferFrom`. This is sufficient, because a payment once received and distributed in the `safeTransferFrom` function will be removed from the mapping.
+* A mapping for listing NFTs to be sold
+
+### Royalty Account Functions
+
+Definitions and interfaces for the Royalty Account RUD (Read-Update-Delete) functions. Because the RA is created in the minting function, there is no need to have a function to create a royalty account separately.
```

Comment:
Is this section missing the actual "definitions and interfaces" or are the other headers at the wrong level?

Reply:
corrected ... I hope